docs/_docs/configuration/menu-hjson.md

```diff
@@ -362,3 +363,188 @@ newUserApplicationPre: {
     // note that the rest of this menu is omitted for clarity
 }
 ```
+
+## Case Study: Manual Encoding Selection
+
+Enigma½ tries to automatically determine the proper encoding for a client when it connects. Unfortunately, there are cases where the wrong encoding can be selected, resulting in terminal programs that are not supported. If your user base contains users that would like to connect with unsupported clients, one solution is to offer manual encoding selection.
+
+A new system method has been added to support this, @systemMethod:setClientEncoding.
+
+### Simple example
+
+A basic config to use this could look something like the following:
+
+
+```hjson
+telnetConnected: {
+    art: CONNECT
+    next: clientSelectEncoding
+    config: { nextTimeout: 1500 }
+}
+
+clientSelectEncoding: {
+    art: CLTSEL.ASC
+    next: matrix
+    config: { interrupt: realtime }
```

Comment:
The interrupt probably doesn't buy much here since this would be a unauthenticated user -- with that said, we probably *don't* want any sorts of interruptions here if that were possible.

Reply:
Removed the interrupt, thanks!